src/xpk/commands/info.py

```diff
@@ -233,6 +222,8 @@ def run_kueuectl_list_clusterqueue(args: Namespace) -> str:
     kueuectl localqueue formatted as json string
   """
   command = 'kubectl kueue list clusterqueue -o json'
+  if args.namespace != '':
+    command += f' --namespace {args.namespace}'
```

Comment:
I wonder if this is necessary since ClusterQueue is an cluster-scoped object so it does not belong to any namespace